static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -71,6 +67,72 @@ define([
         ],
         detect;
 
+    var breakpointNames = breakpoints.map(getBreakpointName);
+
+    var currentBreakpoint;
+    var currentTweakpoint;
+
+    init(window);
+
+    function init(win) {
+        if ('matchMedia' in win) {
+            initMediaQueryListeners(win);
+        } else {
+            updateBreakpoints.call(win);
+            win.addEventListener('resize', debounce(updateBreakpoints, 200));
```

Comment:
we have a few of these debounced now, should we turn this into an app-wide throttled resize, like with scroll?